Source/src/components/ComponentDirLight.cpp

```diff
@@ -60,6 +60,7 @@ void Hachiko::ComponentDirLight::Load(const YAML::Node& node)
 void Hachiko::ComponentDirLight::DrawGui()
 {
     ImGui::PushID(this);
+    DebugDraw();
```

Comment:
why are we calling the debug draw method here? it's supposed to be called from the game object lifecycle and dependent on a setting, since we can turn debug draw on or off

Reply:
The method was never called because we only make debug draw calls if debug draw is enabled. If u use it here the calls would be duplicated when debug draw is enabled.  I think we should not add these debug draw calls

Reply:
Oh yes, my fault, you are right.